### DISCUSSION TITLE
Is there a way to make the bot create a sticker only when it receives a command?

### DISCUSSION THREAD

**Discussion by Adjord** [kaiserdj/Stickerworld#41] [Q&A] [answered]
Instead of turning every image into a sticker, make it only turn if the image sent has !sticker as a message, for example. Is that possible?

**Comment by kaiserdj** [comment, marked as answer]
I am adding an option to activate the creation of the sticker by a command, in the next version this improvement will be included

**Reply by Adjord** [reply]
Ok thank you

**Comment by kaiserdj** [comment]
Improvement included in the project: https://github.com/kaiserdj/Stickerworld/commit/111fdbe03d7c3054fbeef3922bbe150aea8caad0
